mslib/utils/airdata.py

```diff
@@ -43,22 +43,30 @@
 if not os.path.exists(os.path.join(OSDIR, "downloads", "aip")):
     os.makedirs(os.path.join(OSDIR, "downloads", "aip"))
 
-_airspaces = []
-_airports = []
-_airports_mtime = 0
-_airspaces_mtime = {}
-_airspace_url = "https://storage.googleapis.com/29f98e10-a489-4c82-ae5e-489dbcd4912f"
-_airspace_download_url = "https://storage.googleapis.com/storage/v1/b/29f98e10-a489-4c82-ae5e-489dbcd4912f/o/" \
-                         "{}_asp.xml?alt=media"
-# Updated Dec 01 2022
-_airspace_cache = [('ad_asp.xml', '377'), ('ae_asp.xml', '110238'), ('af_asp.xml', '377'), ('ag_asp.xml', '377'),
-                   ('ai_asp.xml', '377'), ('al_asp.xml', '11360'), ('am_asp.xml', '377'), ('ao_asp.xml', '377'),
-                   ('aq_asp.xml', '377'), ('ar_asp.xml', '975402'), ('as_asp.xml', '377'), ('at_asp.xml', '1059077'),
-                   ('au_asp.xml', '9667669'), ('aw_asp.xml', '377'), ('ax_asp.xml', '377'), ('az_asp.xml', '377'),
-                   ('ba_asp.xml', '76717'), ('bb_asp.xml', '377'), ('bd_asp.xml', '377'), ('be_asp.xml', '462667'),
-                   ('bf_asp.xml', '377'), ('bg_asp.xml', '440690'), ('bh_asp.xml', '68519'), ('bi_asp.xml', '377'),
-                   ('bj_asp.xml', '377'), ('bl_asp.xml', '377')
-                 ]
+
+class Global_Constants:
```

Comment:
this is from the name what it is. If you want to describe for whatit  is used do add a doc string. This is then also shown in pycharm etc.

As almost all data inside is prefixed by airspace why not naming it like Airspace and reduce the prefixes? 
airspaces = [] may become data = []

Reply:
I'll get it done.